src/communication/request_client_server.hpp

```diff
@@ -433,16 +450,7 @@ namespace cubcomm
 
     er_log_send_request (chn, static_cast<int> (msgid), packer_current_size);
 
-    // NOTE: no htonl here; integer value sent as a stream of bytes
-    const css_error_code css_size_err = chn.send (
-	reinterpret_cast<const char *> (&packer_current_size), sizeof (packer_current_size));
-    if (css_size_err != NO_ERRORS)
-      {
-	er_log_send_fail (chn, css_size_err);
-	return css_size_err;
-      }
-
-    const css_error_code css_err = chn.send (send_ext_block.get_ptr (), packer.get_current_size ());
+    const css_error_code css_err = chn.send (send_ext_block.get_ptr (), packer_current_size);
```

Comment:
Nit. 
I don't think `packer_current_size` is needed anymore since it's for sending the size, which requires its address.
```suggestion
    const css_error_code css_err = chn.send (send_ext_block.get_ptr (), packer.get_current_size ());
```

Reply:
I know, but it is more explicit like this. Helps for debug/cores.

Reply:
I've got it!